examples/self_managed_node_group/main.tf

```diff
@@ -146,6 +146,37 @@ module "eks" {
       }
     }
 
+    efa = {
+      min_size     = 1
+      max_size     = 2
+      desired_size = 1
+
+      # aws ec2 describe-instance-types --region eu-west-1 --filters Name=network-info.efa-supported,Values=true --query "InstanceTypes[*].[InstanceType]" --output text | sort
+      instance_type = "c5n.9xlarge"
```

Comment:
Looks like the cheapest instance type among the most expensive ones! That's what we need in examples :)

Reply:
ya I know 😅 , I might end up commenting this out and putting a note to check periodically